src/ripple/protocol/STValidation.h

```diff
@@ -25,15 +25,20 @@
 #include <ripple/protocol/PublicKey.h>
 #include <ripple/protocol/STObject.h>
 #include <ripple/protocol/SecretKey.h>
+#include <cassert>
 #include <cstdint>
 #include <functional>
 #include <memory>
 
 namespace ripple {
 
 // Validation flags
-const std::uint32_t vfFullyCanonicalSig =
-    0x80000000;  // signature is fully canonical
+
+// This is a full (as opposed to a partial) validation
+constexpr std::uint32_t vfFullValidation = 0x00000001;
+
+// The signature is fully canonical
+constexpr std::uint32_t vfFullyCanonicalSig = 0x80000000;
```

Comment:
May be have a sense to use prefix 'u' because 0x80000000 is too big?

Reply:
Hmm, I think that's safe, but let me ask the gurus: @HowardHinnant, @scottschurr, any suggestions here?

Reply:
It's safe as is, but I like using the `u` suffix when initializing unsigned.  But I also alphabetize my socks.  So you can't always go by me. :-)

Reply:
:socks:

Reply:
🤣 All of my socks are identical so I don't have to alphabetize them.  I also never need to match them.